nuitka/build/SconsUtils.py

```diff
@@ -469,7 +469,7 @@ def addClangClPathFromMSVC(env):
 
 
 def isGccName(cc_name):
-    return (
+    return "clang" not in cc_name and (
```

Comment:
that one is rather strange, need to check what you saw

Reply:
To avoid things like:

```python
from nuitka.build.SconsUtils import isGccName

isGccName('gcc')  # True

isGccName('clang')  # False

isGccName('g++')  # True
isGccName('clang++')  # ! -> True
```